lib/Conversion/TorchToStablehlo/Basic.cpp

```diff
@@ -1412,6 +1412,40 @@ LogicalResult ConvertAtenOp<AtenPowTensorTensorOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenUniformOp>::matchAndRewrite(
+    AtenUniformOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+  Value self = adaptor.getSelf();
+  Value generator = adaptor.getGenerator();
+  Location loc = op.getLoc();
+
+  if (!generator.getType().isa<Torch::NoneType>())
+    return rewriter.notifyMatchFailure(
+        op, "The generator has to be None because only global default "
+            "generator is supported");
+
+  auto elements = self.getType().cast<RankedTensorType>().getShape();
+  for (auto element : elements) {
```

Comment:
Suggest to use `if (llvm::any_of(elements, [](int64_t dim) { return dim == ShapedType::kDynamic;}` ?